modules/sdk-coin-dot/src/dot.ts

```diff
@@ -276,7 +513,7 @@ export class Dot extends BaseCoin {
   }
 
   async verifyTransaction(params: VerifyTransactionOptions): Promise<boolean> {
-    return true;
+    throw new MethodNotImplementedError();
```

Comment:
Im not 100% sure on this but I think if we throw an error here then it will break OVC which is a piece of software we use to sign txns offline. Lets keep this as true and add a TODO with a ticket number to implement this.